master/buildbot/changes/bitbucket.py

```diff
@@ -159,6 +164,19 @@ def _processChanges(self, result):
                     revision = commit_json['hash']
                     revlink = commit_json['links']['html']['href']
 
+                    # Retrieve the list of added/modified files in the commit
+                    from unidiff import PatchSet
```

Comment:
If we add unidiff to requirements list in setup.py then we can import it at the top of the file as usual.

Reply:
I've also removed the check from the `__init__` method.